<launch>


  <node pkg="map_compression" type="mapCutterTraj" name="mapCutterTraj" output="screen">


	
	<!--For Localization Test-->

	<param name="loadMapName" value="/home/yh/mapModel/2019/01.10/wholeMap_park.ply"/>
	<param name="loadTrajName" value="/home/yh/mapModel/2019/01.10/park_pose.txt"/>
	<param name="keepIndexName" value="/home/yh/mapModel/2019/01.10/keep_park_all.txt"/>
	<param name="saveTrainName" value="/home/yh/mapModel/2019/06.27/split_1.ply"/>
	<param name="saveTestName" value="/home/yh/mapModel/2019/06.27/split_2.ply"/>
	<param name="saveCloudName" value="/home/yh/mapModel/2019/06.27/nouse.ply"/>
	
	<param name="cutPoint0" value="0"/>	
	<param name="cutPoint1" value="1140"/>	
	<param name="cutPoint2" value="1141"/>	
	<param name="cutPoint3" value="1162"/>



  </node>

</launch>
